examples/pass-run-time-values-to-tools.ipynb

```diff
@@ -67,52 +72,171 @@
+   "id": "1c2d0de0-0f3e-4bbe-b0b6-cc0f70b11993",
    "metadata": {},
    "source": [
-    "## Set up the model\n",
-    "\n",
-    "Now we need to load the chat model we want to use.\n",
-    "Importantly, this should satisfy two criteria:\n",
-    "\n",
-    "1. It should work with messages. We will represent all agent state in the form of messages, so it needs to be able to work well with them.\n",
-    "2. It should work with OpenAI function calling. This means it should either be an OpenAI model or a model that exposes a similar interface.\n",
-    "\n",
-    "Note: these model requirements are not requirements for using LangGraph - they are just requirements for this one example.\n"
+    "If we look at the input schemas for these tools, we'll see that user_id is still listed:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "id": "892b54b9-75f0-4804-9ed0-88b5e5532989",
+   "execution_count": 64,
+   "id": "1092929b-c939-4b2a-9f9c-e725b0e34af2",
    "metadata": {},
-   "outputs": [],
-   "source": ["from langchain_openai import ChatOpenAI\n\n# We will set streaming=True so that we can stream tokens\n# See the streaming section for more information on this.\nmodel = ChatOpenAI(temperature=0, streaming=True)"]
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'title': 'get_contextSchema',\n",
+       " 'description': 'Get context on the question.',\n",
+       " 'type': 'object',\n",
+       " 'properties': {'question': {'title': 'Question',\n",
+       "   'description': 'The user question',\n",
+       "   'type': 'array',\n",
+       "   'items': {'type': 'string'}},\n",
+       "  'state': {'title': 'State', 'type': 'object'}},\n",
+       " 'required': ['question', 'state']}"
+      ]
+     },
+     "execution_count": 64,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "get_context.get_input_schema().schema()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e346a26e-e00b-48e5-82c5-c930ea6084a4",
+   "metadata": {},
+   "source": [
+    "But if we look at the tool call schema, which is what is passed to the model for tool-calling, user_id has been removed:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "id": "3912bb51-3107-4335-a659-021c5d89fb37",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'title': 'get_context',\n",
+       " 'description': 'Get context on the question.',\n",
+       " 'type': 'object',\n",
+       " 'properties': {'question': {'title': 'Question',\n",
+       "   'description': 'The user question',\n",
+       "   'type': 'array',\n",
+       "   'items': {'type': 'string'}}},\n",
+       " 'required': ['question']}"
+      ]
+     },
+     "execution_count": 65,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "get_context.tool_call_schema.schema()"
+   ]
```

Comment:
not from your change, but could you please update `StatefulGraph` in the cell below to `StateGraph`?